application/feature/characters/core/api/src/main/java/com/stepango/blockme/feature/characters/core/api/data/mapper/ICharacterMapper.kt

```diff
@@ -0,0 +1,11 @@
+package com.stepango.blockme.feature.characters.core.api.data.mapper
+
+import com.stepango.blockme.core.network.library.response.BaseResponse
+import com.stepango.blockme.feature.characters.core.api.data.response.CharacterResponse
+import com.stepango.blockme.feature.characters.core.api.domain.model.ICharacter
+
+interface ICharacterMapper {
```

Comment:
I can't use Mapper<> from :common.utils in this api module

Reply:
What is the purpose of exposing the mapper in API module?

Reply:
You shouldn't use CharacterMapper as additional interface for api target. This logic need to place in the impl target as "inner" behavior for converting models. 

In the api targets better provide for outer dependecies:
- interfaces for domain models
- interfaces for UseCase's
- interfaces for Repositories